src/test/java/com/google/ehub/data/FavoriteItemDatastoreTest.java

```diff
@@ -0,0 +1,99 @@
+package com.google.ehub.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class FavoriteItemDatastoreTest {
+  private static final String FAVORITE_ITEM_KIND = "favoriteItem";
+  private static final String USER_EMAIL_PROPERTY_KEY = "userEmail";
+  private static final String ITEM_ID_PROPERTY_KEY = "itemId";
+
+  private static final String USER_EMAIL = "bryan@gmail.com";
+  private static final Long ITEM_ID = 1L;
+
+  private final FavoriteItemDatastore favoriteItemDatastore = FavoriteItemDatastore.getInstance();
+  private final DatastoreService datastoreService = DatastoreServiceFactory.getDatastoreService();
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void init() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
+  public void addFavoriteItemToDatastore_EntityGetsAddedWithValidKindAndProperties() {
```

Comment:
nit: after underscore, start again with lowercase

addFavoriteItemToDatastore_entityGetsAddedWithValidKindAndProperties

Reply:
I renamed the test method calls to use lowercase after underscore in last commit.